Code/GraphMol/FileParsers/MolWriters.h

```diff
@@ -14,10 +14,20 @@
 
 #include <RDGeneral/types.h>
 
-#include <string>
 #include <iostream>
+#include <memory>
+#include <string>
```

Comment:
memory is required because of the shared pointers in the `MolWriter` class. Strings were already there, and the header was being pulled in by the ROMol.h include. I just made it implicit so that it doesn't depend on another include.